keel-web/src/main/kotlin/com/netflix/spinnaker/keel/apidocs/DeliveryArtifactModelConverter.kt

```diff
@@ -0,0 +1,14 @@
+package com.netflix.spinnaker.keel.apidocs
+
+import com.netflix.spinnaker.keel.api.artifacts.DebianArtifact
+import com.netflix.spinnaker.keel.api.artifacts.DeliveryArtifact
+import com.netflix.spinnaker.keel.api.artifacts.DockerArtifact
+import org.springframework.stereotype.Component
+
+@Component
+class DeliveryArtifactModelConverter : SubtypesModelConverter<DeliveryArtifact>(DeliveryArtifact::class.java) {
```

Comment:
I thought I saw this in the previous PR? 🤔

Reply:
nope, just resource specs and constraints